### ISSUE TITLE
Možnost administrovat doporučené dokumenty z admin módu klienta

### ISSUE THREAD

**Issue by annie-cz** [ceskaexpedice/kramerius-web-client#707]
Ještě by možná stálo za to přidat do aktuální administrace i možnost označit z klienta některý dokument jako doporučený pro hlavní stránku. Částečně se tomu věnuje issue: https://github.com/ceskaexpedice/kramerius-admin-client/issues/11, navazuje zde na #645 :)

**Comment by zabak** [comment]
To by šlo dělat přes seznamy - vytvoříš si seznamy které chceš dát na hlavní stránku. Ve správci seznamů se ti pokud jsi správce objeví tlačítko sdílet do admina a tím by se seznam načetl do admin rozhraní (resp. jen by se načetl jednorázově obsah toho seznamu, kdybys chtěla udělat změnu na hlavní stránce, musel by se ten seznam nasdílet adminovi znova).  Tak nějak?

**Comment by annie-cz** [comment]
To nezní jako špatné řešení. :)

**Comment by honza-rychtar** [comment]
Nejdříve celou tu konfiguraci musíme celou tu konfiguraci přesunout do jádra/API.
Teď je to celé v konfiguraci klienta.

Mělo by to být součásti balíku přesunu dalších věcí do jádra, zejména licencí. @zabak ?

**Comment by zabak** [comment]
Ano, souhlasím. Založil jsem #711 kam by se měly takové požadavky koncentrovat.
